app/models/user.rb

```diff
@@ -7,6 +7,6 @@ class User < ApplicationRecord
 
   # default is 0 (:user)
   enum role: [:user, :superadmin, :admin, :lab_admin]
-  validates :given_name, :last_name, :institutional_id, presence: true
+  validates :given_name, :institutional_id, presence: true
```

Comment:
por qué dejar de validar last name?

Reply:
Well porque no estoy seguro de que valga la pena validar el last name. Consider this: qué last name le pones a administration accounts? (Admin, superadmin, Innovaction) etc